### PR TITLE
#1748 misc dispensing components

### LINKED ISSUE #1748: Misc components needed for Prescriptions
## Is your feature request related to a problem? Please describe.

In order to do #1738 there are a couple more components required

## Describe the solution you'd like

components:
- A circular icon component because circular icons are nice
- A menu icon added to `icons.js`
- 3 layout components: a stepper + a label, a list of details for an item, a dropdown menu + label + burger menu button

this is to create a row here:
image

which would be broken down like:
image

## Implementation

As above

## Describe alternatives you've considered

N/A

## Additional context

N/A

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #1748 

## Change summary

- Adds quite a few components..

`CircleButton`:
image

`StepperRow`:
image

`DetailRow`:
image

`DropdownRow`:
image

None are really 100% right, but in general I think they're minor style adjustments that can be iterated on when pulling together

## Testing

N/A

### Related areas to think about

- Really need better ways to create things .. more smaller components to use I think.. like `<FlexRow flex={2}> ...` which is just a `View` with `flexDirection: "row", flex: flex` .. same with colums etc!

### CODE REVIEW

File: src/widgets/DetailRow.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * Each SimpleLabel has a label and some other text.
+ *
+ * @prop {Array} details Array of objects in the form {label, text}
+ */
+export const DetailRow = ({ details }) => {
+  const Details = React.useCallback(
+    () =>
+      details.map(({ label, text }) => (
+        <View>
```
Comment (mark-prins) [inline src/widgets/DetailRow.js]: should have a `Key` on the `View`

File: src/widgets/icons.js [hunk 2]
```diff
@@ -86,3 +86,13 @@ export const ChevronRightIcon = () => (
 );
 
 export const FavouriteStarIcon = () => <FAIcon name="star-o" color={SUSSOL_ORANGE} size={20} />;
+export const BurgerMenuIcon = () => <EntypoIcon name="menu" color={SUSSOL_ORANGE} size={30} />;
```
Comment (mark-prins) [inline src/widgets/icons.js]: three different icon libraries?! does that mean we have an inconsistent look or something?

File: src/widgets/DetailRow.js (hunk 1 above)
Comment (joshxg) [reply src/widgets/DetailRow.js]: e43d55f892a0181d970a74864b87b03a800c0abf

File: src/widgets/icons.js (hunk 2 above)
Comment (joshxg) [reply src/widgets/icons.js]: I've made this `icons` file to try and get it more consistent.. previously we had 3 different magnifying glass icons! But yes.. and I think it's 6 icon libraries.... :)

**Pull request merged**